src/core/Form/TextInput/TextInput.baseStyles.tsx

```diff
@@ -17,21 +19,29 @@ export const baseStyles = withSuomifiTheme(
     flex-direction: column;
 
     & .fi-text-input_statusText_span {
-      ${theme.typography.bodySemiBoldSmall}
+      font-size: 14px;
```

Comment:
Removing the typography token also removes the font-face definition. Does this allow font-face bleeding into status text from outside or will this be nested within a block that has more specific typography rules?

Reply:
Good point. Better safe than sorry. Now fixed.